src/styles/cards/card.scss

```diff
@@ -0,0 +1,89 @@
+.card {
+  width: $card-width;
+  border-radius: 5px;
+  border: 1px solid $border-color-card;
+  justify-content: center;
+  display: block;
```

Comment:
The 'display: block' property is not necessary for the '.card' class. By default, divs are block-level elements. However, if you want to use flex properties like 'justify-content' and 'flex-direction', you should use 'display: flex' instead of 'display: block'.